src/Commands/core/CliCommands.php

```diff
@@ -82,9 +81,12 @@ public function cli(array $options = ['version-history' => false, 'cwd' => self:
         $shell = new Shell($configuration);
 
 
-        // Register the assertion handler so exceptions are thrown instead of errors
-        // being triggered. This plays nicer with PsySH.
-        Handle::register();
+        // Register the assertion handler so exceptions are thrown instead of
+        // errors being triggered. This plays nicer with PsySH. Since we're
+        // using exceptions, turn error warnings off.
+        assert_options(ASSERT_EXCEPTION, true);
+        assert_options(ASSERT_WARNING, false);
```

Comment:
It should have wrapper for PHP 8.3 https://php.watch/versions/8.3/assert-multiple-deprecations